source/stylesheets/refills/_hover-tile-animation.scss

```diff
@@ -40,21 +28,16 @@
   .hover-tile {
     @include transition(all, 0.2s ease-in-out);
     background: inherit;
-    color: white;
     height: inherit;
     overflow: hidden;
     padding: $base-spacing;
   }
 
   .hover-tile-hidden {
-    background: transparentize(#000, 0.5);
-
-    p {
-      color: transparentize(#fff, 0.3);
-    }
+    background: $hover-tile-background;
 
     h4 {
-      margin-bottom: 0.5em;
+      margin: 0 0 0.5em 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 0.5em` instead of `0 0 0.5em 0`